src/neighbors.cc

```diff
@@ -1148,22 +1149,33 @@ peerDNSConfigure(const ipcache_addrs *ia, const Dns::LookupDetails &, void *data
 }
 
 static void
-peerRefreshDNS(void *data)
+peerDnsRefreshStart()
 {
-    if (eventFind(peerRefreshDNS, nullptr))
-        eventDelete(peerRefreshDNS, nullptr);
+    for (const auto &p: CurrentCachePeers())
+        ipcache_nbgethostbyname(p->host, peerDNSConfigure, p.get());
 
+    /* Reconfigure the peers every hour */
```

Comment:
Let's drop this misleading comment. This self-documented code is _not_ reconfiguring cache_peers. The only problem here is that 3600.0 constant lacks "this is an hour" annotation, but that will be fixed when we migrate this code to std::chrono.

```suggestion
```

> that will be fixed when we migrate this code to std::chrono

I do not think we should migrate in this PR, but we will migrate if others insist.